lib/dradis/plugins/netsparker/importer.rb

```diff
@@ -19,20 +19,35 @@ def import(params={})
       end
 
       @doc.xpath('/netsparker/vulnerability').each do |xml_vuln|
-        process_vulnerability(xml_vuln)
+        # Create Nodes from the <url> tags
```

Comment:
If in the future we're going to want to have all plugins create nodes by IP address, maybe we should start doing this for new plugins like this one?

I'm OK with what we have, but it seems that in the future we'd like the Node's label to be the IP address, and a node "hostname" property to be the value of the host from the `./url` element.